src/validator/CreditCardVendor.java

```diff
@@ -0,0 +1,32 @@
+package validator;
+
+import java.util.ArrayList;
+
+public class CreditCardVendor {
+	private String name;
```

Comment:
Vendor names should be kept as Enum.
(Vendors are loaded from a file, but I would still model them as an Enum - there is a limited number of card vendors and having their names as some constant will help avoid simple mistakes.